gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/PublisherSettings.java

```diff
@@ -412,4 +285,196 @@ public long countBytes(PublishRequest request) {
           return request.getSerializedSize();
         }
       };
+
+  public static class Builder extends ServiceApiSettings.Builder {
+    private final ImmutableList<ApiCallSettings.Builder> methodSettingsBuilders;
+
+    private SimpleCallSettings.Builder<Topic, Topic> createTopicSettings;
+    private BundlingCallSettings.Builder<PublishRequest, PublishResponse> publishSettings;
+    private SimpleCallSettings.Builder<GetTopicRequest, Topic> getTopicSettings;
+    private PageStreamingCallSettings.Builder<ListTopicsRequest, ListTopicsResponse, Topic>
+        listTopicsSettings;
+    private PageStreamingCallSettings.Builder<
+            ListTopicSubscriptionsRequest, ListTopicSubscriptionsResponse, String>
+        listTopicSubscriptionsSettings;
+    private SimpleCallSettings.Builder<DeleteTopicRequest, Empty> deleteTopicSettings;
+
+    private static final ImmutableMap<String, ImmutableSet<Status.Code>> RETRYABLE_CODE_DEFINITIONS;
```

Comment:
You may want to be aware of #883.
Summary: we think it makes sense to retry when:
idempotent AND error code suggests it makes sense to retry
OR
error code suggests that it makes sense to retry and that request was rejected

An example of the latter case would be a short burst quota exceeded.